package.json

```diff
@@ -19,5 +19,11 @@
     "@mui/icons-material": "^5.15.13",
     "@mui/material": "^5.15.13",
     "js-cookie": "^3.0.5"
+  },
+  "devDependencies": {
+    "@typescript-eslint/eslint-plugin": "^7.3.1",
+    "@typescript-eslint/parser": "^7.3.1",
+    "eslint": "^8.56.0",
+    "eslint-plugin-react": "^7.34.1"
```

Comment:
Since we should remove the eslint file from the root we should also remove these 2 eslint dependencies.
npm uninstall eslint eslint-plugin-react.

Reply:
that was fixed in another commit, and i also did the npm uninstall. fixed